server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -268,10 +277,12 @@ private void handleSuggestion(JSONObject accusation, Player suggester) {
 			suggestion.put(Constants.ACCUSED, theAccused.toString());
 			suggestion.put(Constants.WEAPON, theWeapon.toString());
 			suggestion.put(Constants.ROOM, suggestedRoom.toString());
+			handleSimpleEvent(suggester, EventType.PREVENT_TURN_END);
 			notifyPlayers(suggestion);
 			JSONObject move = makeMoveNotification(theAccused, suggestedRoom);
 			
 			board.movePiece(theAccused, suggestedRoom);
+			allowMovedPlayerToSuggest(theAccused);
```

Comment:
Nope because the buttons are greyed out. But just in case I added some logic to check if it's your turn before suggesting.